### PR TITLE
2 implement onespy

### LINKED ISSUE #2: Implement `ones.py`

### PR DESCRIPTION
**Pull request by Sam4823**
Hello pls pls pls pls pls pull

### CODE REVIEW

**Comment by SepehrMoalemi** [comment]
too slow. dont use for loops
git better
`np.sum(dice[dice == 1])`

File: src/yahtzee/ones_box.py [hunk 1]
```diff
@@ -16,4 +16,8 @@ def ones(dice: np.ndarray) -> int:
     int :
         Score.
     """
-    raise NotImplementedError()
```
Comment (andrewipstirling) [inline src/yahtzee/ones_box.py:19]: perfect.

File: tests/test_ones_box.py [hunk 2]
```diff
@@ -2,3 +2,8 @@
 
 import numpy as np
 import yahtzee.ones_box
+
+def test_ones():
```
Comment (andrewipstirling) [inline tests/test_ones_box.py:6]: really good sam.

**Review by andrewipstirling — APPROVED** [review]
sam... where do i start? never, in my 23 years have I read code this clean... congratulations. approved.

**Pull request merged**